### DISCUSSION TITLE
Accessing the Voice of the Customer data programmatically

### DISCUSSION THREAD

**Discussion by EhsanMalik360** [amzn/selling-partner-api-models#4129] [Q&A]
Hi Everyone!

I want to access the Voice of the Customer Data about Customer Experience (CX) which is present under the performance tab in Amazon Seller Central.

I want to fetch this data programmatically using API or connector. The main purpose is to automatically fetch this data in a Google Sheet on daily basis without any human intervention.

The main data points I want to fetch are;
1. ASIN
2. SKU
3. Date
4. OrderID
5. Comments
6. Ratings.

 I wanted to check with the community:

Is there an endpoint within the SP-API that provides access to this Voice of the Customer data?
If not, does anyone have experience with a third-party service that could automate this process?
We're looking for a solution that can help us fetch our data without manual intervention.

Thank you in advance for your guidance and suggestions!

Best regards,
Ehsan

**Comment by mafge** [comment]
Hi @EhsanMalik360,
thanks for reaching out!
You can have a look at [Performance Reports](https://developer-docs.amazon.com/sp-api/docs/report-type-values-performance) and check if there is a report that has the data you are looking for. Hope that helps!
Best,
Marc
Selling Partner Developer Services

**Reply by EhsanMalik360** [reply]
> Hi @EhsanMalik360, thanks for reaching out! You can have a look at [Performance Reports](https://developer-docs.amazon.com/sp-api/docs/report-type-values-performance) and check if there is a report that has the data you are looking for. Hope that helps! Best, Marc Selling Partner Developer Services

Thanks for your reply,

I have checked and found that this report is most relevant. 
https://developer-docs.amazon.com/sp-api/docs/report-type-values-performance#flat-file-feedback-report

But it doesn't have the ASIN and SKU.

Also, can you please confirm that this report is regarding the customer experience data that is available in Voice of the Customer dashboard, rather than general comments or ratings about some orders?

Thanks

**Reply by mafge** [reply]
@EhsanMalik360 The report you are mentioning contains ratings of the seller itself and not ratings of products ordered from a seller. Actually by looking further into the API I don't think there is a possibility to get the data you want. Please also note that reports from the API differ from the ones offered by Seller Central.
